src/solvers/prop/decision_procedure.h

```diff
@@ -54,6 +60,7 @@ class decision_proceduret
 
   /// Return value of literal \p l from satisfying assignment.
   /// Return tvt::UNKNOWN if not available
+  /// This will go away, use get instead.
```

Comment:
Mark as deprecated?

Reply:
Will remove the usage first (not a lot).